ddsrecorder_participants/src/cpp/recorder/mcap/McapHandler.cpp

```diff
@@ -130,6 +125,7 @@ void McapHandler::add_schema(
 
     if (configuration_.ros2_types)
     {
+        // NOTE: Currently ROS2 types are not supported, change when supported
```

Comment:
I would remove this note, what is currently not supported is ROS 2 endpoints sending dynamic types, but what this feature actually means is "store in ROS 2 format".